### PR TITLE
OpenCL: Implement freeImpl()

### PR DESCRIPTION
**Pull request by linehill**

### CODE REVIEW

File: src/backend/OpenCL/CHIPBackendOpenCL.cc [hunk 1]
```diff
@@ -470,19 +470,23 @@ CHIPKernelOpenCL::CHIPKernelOpenCL(const cl::Kernel &&ClKernel,
 // CHIPContextOpenCL
 //*************************************************************************
 
-void CHIPContextOpenCL::freeImpl(void *Ptr) { UNIMPLEMENTED(); };
+void CHIPContextOpenCL::freeImpl(void *Ptr) {
+  size_t Ignored;
+  SvmMemory.free(Ptr, &Ignored);
```
Comment (pvelesko) [inline src/backend/OpenCL/CHIPBackendOpenCL.cc:475]: What is the purpose of the size argument?

Comment (pvelesko) [reply src/backend/OpenCL/CHIPBackendOpenCL.cc:475]: @linehill

Comment (linehill) [reply src/backend/OpenCL/CHIPBackendOpenCL.cc]: No idea what the original author had in mind for the argument. Seems unnecessary so I removed it.

**Comment by linehill** [comment]
Updated: Removed unused size parameter.

**Pull request merged**